templates/workspaces/base/porter.yaml

```diff
@@ -66,8 +66,11 @@ parameters:
     description: "Whether this bundle should register the workspace in AAD"
   - name: create_aad_groups
     type: boolean
-    default: false
-    description: "Whether this bundle should create AAD groups for the workspace app roles"
+    default: true
+    description: "Whether this bundle should create AAD groups for the workspace app roles (required for User Management)"
+  - name: core_api_client_id
```

Comment:
This seem to be an _internal_ param. Should we hide it in the UI (`tre-hidden` css class)?
Also applies to the other workspaces you added the param to.

Reply:
btw, I might have been to quick on this as I didn't confirm if you've added it to template_schema. sorry.